mimic/test/test_images.py

```diff
@@ -0,0 +1,113 @@
+"""
+Tests for :mod:`nova_api` and :mod:`nova_objects` for images.
+"""
```

Comment:
sure, sounds good. I thought I had done that already, but if I had left some, please do so.

Also, lets please merge test_images and test_server_images.